checkout.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: lesson
+subtitle: Checkout Procedure
+email: training-help@software-carpentry.org
+---
+After you have finished the two-day training class,
+you must go through these steps in order to complete your training and be certified:
+
+1.  Submit a [small change to one of our lessons](#lesson-change).
+2.  Take part in an online [discussion session](#discussion-session).
+3.  Teach a short [demonstration lesson](#demo-lesson) online.
+
+We would like to get everyone through this process within two months of training,
+so please get started as soon as you can.
```

Comment:
I'm not so worried about shortening this - trainees will be reading it on their own time, and if you think of it as a FAQ meant to anticipate and answer their questions, it really does have to spell things out in detail.  (Most of the points are taken directly from questions we've had and answers we've sent...)